### DISCUSSION TITLE
Support for superscript and subscript

### DISCUSSION THREAD

**Discussion by francis6425** [ueberdosis/tiptap#813] [General Questions]
Hi,
I'm new to tiptap and like it alot. However it seems like it is not supporting  superscript and subscript.
Can it be added ?

**Comment by hanspagel** [comment]
Sure! Try the following:

1. Copy the Bold extension (https://github.com/ueberdosis/tiptap/blob/main/packages/tiptap-extensions/src/marks/Bold.js) to your project.
2. Replace `strong` with `sup`, it should look like that: https://gist.github.com/hanspagel/edae39ffee55fd515446702222e06f9d (Warning, I didn’t test this!)
3. Import that Superscript extensions and register it with tiptap `new Editor({extensions: [ new Superscript(), ]})`
4. Add a button to your editor that runs `editor.commands.superscript`.

Hope that helps!

**Reply by hanspagel** [reply]
I published npm packages with extensions for superscript and subscript. 🤓

https://github.com/hanspagel/tiptap-extension-superscript
https://github.com/hanspagel/tiptap-extension-subscript
